src/Services/Export/WB.Services.Export.Host/Controllers/TenantController.cs

```diff
@@ -40,15 +51,50 @@ public async Task<ActionResult> StopTenant([FromQuery] string? tenant = null)
                 tenant = context.Tenant.Name;
             }
             
-            if (string.IsNullOrWhiteSpace(tenant)) return BadRequest("No tenant specified");
+            if (string.IsNullOrWhiteSpace(tenant)) 
+                return BadRequest("No tenant specified");
+            
+            this.logger.LogCritical("Export service tenant {tenant} data deleted due to the request", tenant);

```

Comment:
Please change wording to remove confusion. Tenant data was requested to be deleted but not yet deleted.